src/util/apiUtils.js

```diff
@@ -317,10 +320,22 @@ export function getAuthHeaders(request) {
 	};
 }
 
+export function getLanguageHeader(request) {
+	const cookieLang = getCookieLang(request);
+	const headerLang = request.headers['accept-language'];
+	const acceptLang = cookieLang ?
+		`${cookieLang},${headerLang}` :
+		headerLang;
+	return {
+		'accept-language': acceptLang,
+	};
+}
+
 export function parseRequestHeaders(request) {
 	const externalRequestHeaders = {
 		...request.headers,
```

Comment:
actually, i see even when i change my locale it wasn't passing the write language to accept-language